api-server/README.md

```diff
@@ -0,0 +1,112 @@
+
+In addition to the PostgreSQL, an implementation of a full in-memory storage exists, which we use for testing and as a reference implementation. Hence, when adding a new database implementation, the in-memory implementation can be used as a reference implementation. Our tests ensure that both PostgreSQL and in-memory implementation, through the beautiful abstractions of rust, both arrive to the same result. Any additional implementation can be added to the same test suite.
+
+## How to run
+
+In the following we present the minimal requirements to run the API server in action. In this example, we will be using the testnet. Replace every `testnet` with `mainnet` for the mainnet.
+
+### Make sure you have a node running
+
+If you don't have a node running, the API server won't have a source, from which it can read block data. To run the node, please consult the [the main readme file of the repo](/README.md).
+
+### How to run the database
+
+To run the database, you can use use docker or podman. For simplicity, so that root isn't needed, podman can be used:
+
+```
+podman run --detach --rm --name MintlayerAPIServerDB -e POSTGRES_HOST_AUTH_METHOD=trust -e POSTGRES_DB=mintlayer-testnet -p 127.0.0.1:5432:5432 docker.io/library/postgres
+```
+
+Notice a few things:
+
+1. This command doesn't need root
+2. The database is running in trust mode, which means no username or password required
+3. The command line argument `--rm` is used, so that the container will be deleted on exit
+4. The database name is `mintlayer-testnet`, which is a standard used in the form of `mintlayer-`, followed by the network name (mainnet, testnet, etc)
+5. The port binding `127.0.0.1:5432:5432` doesn't allow external computers in the network to connect. Binding to `0.0.0.0` has security implications that are out of the scope of this documentation. Please make sure to have proper network security when running a database.
+6. There's no volume set. Meaning: Once the container is stopped and removed, all the data in the database will be lost. Please consult the documentation of the container to learn how to preserve the database data.
+
+Please understand that this is just a minimal example, and for real infrastructure, proper security must be considered.
+
+### How to run the blockchain scanner daemon
+
+Assuming the database works as described earlier, the blockchain scanner daemon can be run from the source code using:
+
+```
+cargo run --bin api-blockchain-scanner-daemon --release -- --network testnet
+```
+
+or if you want to run using the executable directly:
+
+```
+api-blockchain-scanner-daemon --network testnet
+```
+
+And this should immediately work, where the blockchain scanner daemon will communicate with the default network RPC network port of the node (13030 for testnet, 3030 for mainnet), and it will also communicate with the database and write the data it finds in the blockchain.
+
```

Comment:
> And this should immediately work, where the blockchain scanner daemon will communicate ...

Sounds a bit off. I suggest to put full stop after `work` and start a new sentence:
"And this should immediately work. The blockchain scanner daemon will communicate..."